.github/workflows/update-changelog.yml

```
@@ -34,6 +34,11 @@ jobs:
 
       - name: Update CHANGELOG.md
         run: |
+          if [ ! -d "${{ steps.extract.outputs.prefix }}" ]; then
+            echo "Directory ${{ steps.extract.outputs.prefix }} does not exist, skipping changelog update"
+            exit 0
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)